cmd/acra-translator/grpc_api/decryptor.go

```diff
@@ -11,25 +11,31 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// DecryptGRPCService represents decryptor for decrypting AcraStructs from gRPC requests.
 type DecryptGRPCService struct {
 	*common.TranslatorData
 }
 
+// NewDecryptGRPCService creates new DecryptGRPCService.
 func NewDecryptGRPCService(data *common.TranslatorData) (*DecryptGRPCService, error) {
 	return &DecryptGRPCService{TranslatorData: data}, nil
 }
 
-var ErrCantDecrypt = errors.New("can't decrypt data")
-var ErrClientIdRequired = errors.New("ClientId is empty")
+// Errors possible during decrypting AcraStructs.
+var (
+	ErrCantDecrypt      = errors.New("can't decrypt data")
+	ErrClientIDRequired = errors.New("ClientID is empty")
```

Comment:
`CliendId` was used because it's name of generated filed with protobuf compiler (cmd/acra-translator/grpc_api/api.pb.go:27). I think better to use field name as is